src/main/java/com/cloudbees/groovy/cps/TransformerConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package com.cloudbees.groovy.cps;
+
+import com.cloudbees.groovy.cps.impl.CpsClosure;
+import org.codehaus.groovy.ast.ClassNode;
+
+/**
+ * Switches that affect the behaviour of {@link CpsTransformer}
+ *
+ * @author Kohsuke Kawaguchi
+ * @see CpsTransformer#setConfiguration(TransformerConfiguration)
+ */
+public class TransformerConfiguration {
+    private ClassNode closureType = new ClassNode(CpsClosure.class);
+
+    public ClassNode getClosureType() {
+        return closureType;
+    }
+
+    public TransformerConfiguration withClosureType(ClassNode closureType) {
```

Comment:
Maybe pass a `Class<? extends CpsClosure>` instead and create the `ClassNode` from it?